software/HeartRateDetection_V2.0/HeartRateDetection_V2.0.ino

```diff
@@ -0,0 +1,230 @@
+// HeartRateDetection_V2.0 - BioAmp EXG Pill
+// https://github.com/upsidedownlabs/BioAmp-EXG-Pill
+// https://github.com/upsidedownlabs/Heart-BioAmp-Arduino-Firmware
+
+// Upside Down Labs invests time and resources providing this open source code,
+// please support Upside Down Labs and open-source hardware by purchasing
+// products from Upside Down Labs!
+
+// Copyright (c) 2021 - 2024 Upside Down Labs - contact@upsidedownlabs.tech
+// Copyright (c) 2021 - 2024 Aryan Prakhar - aryanprakhar1010@gmail.com

```

Comment:
Please add Dev as a contributor here as well.

Reply:
I have add Dev as a contributor for this project